pkg/doc/jsonld/context/contexts.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package context
+
+type contextDocument struct {
+	documentURL string
+	path        string
+}
+
+// contexts contains a list of JSON-LD context documents that should be preloaded into DB.
+var contexts = map[string]contextDocument{
+	"https://w3id.org/security/v1": {
+		documentURL: "https://w3c-ccg.github.io/security-vocab/contexts/security-v1.jsonld",
+		path:        "preload/security_v1.jsonld",
+	},
+
+	"https://w3id.org/security/v2": {
+		documentURL: "https://w3c-ccg.github.io/security-vocab/contexts/security-v2.jsonld",
+		path:        "preload/security_v2.jsonld",
+	},
+
+	"https://w3id.org/security/jws/v1": {
+		documentURL: "https://w3c-ccg.github.io/lds-jws2020/contexts/v1/",
```

Comment:
use the canonical URL (not the redirected one): https://w3id.org/security/jws/v1